src/main.rs

```diff
@@ -30,8 +30,15 @@ static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 #[allow(clippy::cyclomatic_complexity)]
 fn main() {
     let rpc_server_bind_addr = default_rpc_server_bind_addr();
+    let long_version_str: &str = &format!(
+        "{}\nBuilt by {}-mode with {}",
```

Comment:
I don't like the output format because it is not extensible. The format should be entirely re-written even if we attempt to add a commit id of git or else.

To make the format extensible, I prefer to use a separator like the following example in `clap`(https://docs.rs/clap/2.29.0/src/clap/app/mod.rs.html#357):

```
App::new("myprog")
    .long_version(
"v0.1.24
 commit: abcdef89726d
 revision: 123
 release: 2
 binary: myprog")
```